Note how to translate `python-docs-theme` (#1650)
python/devguide: documentation/translations/translating.rst

           print(kw, ":", keywords[kw])

.. _transifex-use:

Transifex
=========

.. important::

   There are many translations in the `python-doc organization on Transifex <tx_>`_,
   some of which, however, are not used or do not have a coordination team.

>    Confirm that a coordination team exists before you begin translation.

Reviewer: `begin translation` doesn't read too well. Let's do either `begin translating` or `start with the translation`.

```suggestion
   Confirm that a coordination team exists before you begin translating.
```

- `Starting with the basics <https://help.transifex.com/en/collections/3441044-starting-with-the-basics>`__:
   A group of documents with basic information.

Within the organization, a project for translating the

> `python-docs-theme <https://github.com/python/python-docs-theme>`_ can also be

Reviewer 2: ```suggestion
:github:`Python Docs Sphinx Theme <python/python-docs-theme>` can also be
```

your languages coordination team.

.. _python-docs-theme-i18n:

> How do I translate ``python-docs-theme``?
> -----------------------------------------

Reviewer 2: ```suggestion
How do I translate Python Docs Sphinx Theme?
--------------------------------------------
```

> The Sphinx theme for the Python documentation supports internationalization.

Reviewer 2: ```suggestion
The Sphinx theme for the Python documentation supports localization.
```

You can translate either on
`Transifex <https://explore.transifex.com/python-doc/python-docs-theme/>`_
(see :ref:`translating on Transifex <transifex-use>` for more information)
or locally by following the steps outlined below.

> To translate locally, clone the ``python-docs-theme``
> `repository <https://github.com/python/python-docs-theme>`_ and run the following

Reviewer 2: ```suggestion
To translate locally, clone the :github:`Python Docs Sphinx Theme repository <python/python-docs-theme>` and run the following
```

commands to generate the PO files. Replace ``LANG`` with the same language code
that is used for the docs translation:

.. code-block:: bash

    python babel_runner.py extract
    python babel_runner.py init -l LANG

The file can then be found at:

.. code-block:: text

    python-docs-theme/locale/LANG/LC_MESSAGES/python-docs-theme.po

After translating, submit your PO file via a pull request to the

> `repository <https://github.com/python/python-docs-theme>`_.

Reviewer 2: ```suggestion
:github:`repository <python/python-docs-theme>`.
```
